### DISCUSSION TITLE
Allow disabling hardware acceleration

### DISCUSSION THREAD

**Discussion by nitinskanwar** [f1multiviewer/issue-tracker#45] [Feature requests] [answered]
I have a multiple monitor setup with my Macbook Pro (Intel) using a Totu hub. When I'm running the Live streams, the videos are not showing up but I can hear the sound in the background. I had a similar issue with Chrome when I watched the race on multiple screens. But after unchecking the hardware acceleration option in Chrome, I was able to get it working. Any ideas as to how I can fix this issue with the multiviewer?

**Comment by f1multiviewer** [comment, marked as answer]
This is available (for a long time actually), in settings :)
